kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -1253,6 +1253,7 @@
                   <home>${websphere.home}</home>
                   <timeout>240000</timeout>
                   <systemProperties>
+                    <org.kie.server.id>test-kie-server</org.kie.server.id>
```

Comment:
I'm not very sure if this is the right way to fix. We want to use a test utility class to set the `serverId`?

Interestingly, this issue doesn't happen with other containers (wildfly, tomcat, weblogic), so probably it has something to do with when `serverId` should be set.

Reply:
I guess websphere doesn't execute ServletContextListener on deployment (so Bootstrap mechanism to provide an arbitrary name doesn't work here), but as this parameter is mandatory, it's a good fix.